src/components/organisms/DataExplorer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Pane } from 'evergreen-ui';
+import DataResultItem from '../molecules/DataResultItem';
+import { useQuery } from '@apollo/client';
+import { METADATA, MetadataEntry, Metadata } from '../../queries/metadata';
+import Loading from '../atoms/Loading';
+import ErrorMessage from '../atoms/ErrorMessage';
+
+const DataExplorer: React.FC = () => {
+    const { data, loading, error } = useQuery<Metadata>(METADATA);
```

Comment:
Can't you use the wrapper you made earlier here?

You're checking for loading, error and data, which is the same as in DataWrapper.

Reply:
I guess you could, but we won’t t be formatting any data like we do in dataWrapper

Reply:
Then you could simply have the mapping function be `e => e`.

Reply:
You will probably have to type the parameters, however, as using generic parameters on the component seems to break your current build CI action.

E.g.

```tsx
(e: Type): Type => e
```

I would argue that it would be better to include the generic parameters on the component, but this requires @blauks to look at the CI action and resolve the issues there.

Reply:
It only breaks the Docker CI, which I do not understand 😀

Reply:
You're not expected to understand it now, but it probably has a logical explanation.

The docker configuration is deterministic, so it should in all theory be reproducible locally -- and thus easier to debug and fix.

Reply:
The CI part shouldn't affect the outcome -- as is the point of docker.

Reply:
DataExplorer will be expanded with search and filter later. Isn’t it better to let DataExplorer handle all this instead of dataWrapper? The styling will also be different for errors and loading so DataWrapper must be changed if we want to use it in DataExplorer.

Reply:
I agree, I think it is best to keep them seperated.